azure-pipelines.yml

```diff
@@ -1,7 +1,7 @@
 trigger:
 - master
-- dev
 - refs/tags/*
+- dev
```

Comment:
Should this be here in the master pipeline?

Reply:
For now.  To remove as part of this PR, we'd have to remove from `dev` branch as well.  Which would result in no build trigger for `dev` branch.

Changing this, as well as fixing the _branch_ var in README.adoc will need to be part of a separate effort.